app/containers/Bookings/constants.js

```diff
@@ -22,6 +22,13 @@ export const offenderImageUrl = imageId => {
   return '/app/images/placeholder/data'
 }
 
+export const offenderFullSizeImageUrl = imageId => {
```

Comment:
I'd probably use a ternary express here. However, it's largely subjective. 
   `export const offenderFullSizeImageUrl  = imageId => imageId ? 
               `/app/full-size-image/${imageId}/data` :
              '/app/full-size-image/placeholder/data'

Reply:
We were just keeping its consistent with the code above that line.